index.js

```diff
@@ -0,0 +1,102 @@
+const playersData = "http://localhost:3000/players"
```

Comment:
Try to use a more descriptive variable name here, 'data' is pretty generic. Maybe `playerUrl`?